exercises/sum-of-multiples/.meta/template.j2

```diff
@@ -0,0 +1,11 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        self.assertEqual({{ case["property"] }}({{ case["input"]["limit"] }}, {{ case["input"]["factors"] }}), {{ case["expected"] }})
```

Comment:
The reason the existing code uses `sum_of_multiples` instead of `sum` is that `sum` is a built-in function. It is against best practices to shadow built-in symbols, so the method was called `sum_of_multiples` instead.

In the case of this exercise, I would suggest the following:

```suggestion
        self.assertEqual({{ case["property"].replace("sum", "sum_of_multiples") }}({{ case["input"]["limit"] }}, {{ case["input"]["factors"] }}), {{ case["expected"] }})
```